packages/core/src/command/consolidated/consolidated-filter-by-date.ts

```diff
@@ -46,16 +44,21 @@ export function filterBundleByDate(bundle: Bundle, dateFrom?: string, dateTo?: s
   if (!dateFrom && !dateTo) return bundle;
   const range = { dateFrom, dateTo };
 
+  // const patient = bundle.entry?.find(entry => entry.resource?.resourceType === "Patient")
```

Comment:
assuming these commented out pieces have to do with one of the WIP TODOs?

Reply:
Yep, to be addressed w/ the age calculation on the next PR